### ISSUE TITLE
Mac installation reports success but doesn't appear in renderers dropdown

### ISSUE THREAD

**Issue by axjxwright** [GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD#165]
Tried both running the supplied .command as well as just calling the python script myself. The output seemed to be what I would expect a successful install to look like: 

/Houdini/Current is symlinked to Houdini18.0.287

```
Installing "hdRpr-Houdini-1.0.0-macOS.tar.gz"
hdRpr will be installed to "/Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources"
Do you agree? [y/n]
y
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Resources/houdini/soho/parameters/HdRprPlugin_Viewport.ds
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/libRadeonImageFilters64.1.4.2.dylib
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/libTahoe64.dylib
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/libRadeonImageFilters64.dylib
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/libRadeonImageFilters64.1.dylib
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/plugInfo.json
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/images/restartRequired.png
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/kernels/gfx803_36.cd.pdb.txt
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/kernels/gfx902_8.cd.updb.txt
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/kernels/gfx900_56.cd.pdb.txt
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/kernels/gfx900_64.cd.pdb.txt
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/kernels/gfx906_60.cd.updb.txt
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/kernels/gfx906_60.cd.pdb.txt
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/kernels/gfx803_64.cd.pdb.txt
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/kernels/gfx906_56.cd.pdb.txt
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/kernels/gfx906_64.cd.pdb.txt
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/esrgan-03x2x32-273866.pb
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/denoise_c3_ldr.pb
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/rif_models.version
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/denoise_c9_ldr.pb
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/denoise_c3_hdr.pb
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/upscale2x_fast.pb
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/denoise_c9_hdr.pb
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr/resources/rif_models/esrgan-05x3x32-278391.pb
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/usd_plugins/hdRpr.dylib
-- Installing: /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/Libraries/libRadeonProRender64.dylib
Successfully installed
```

After launching houdini, RPR isn't visible in the list of renderers.
<img width="156" alt="no-rpr" src="https://user-images.githubusercontent.com/959050/69907418-6b062180-1429-11ea-9985-6ed9cf89cb04.png">

2016 Macbook Pro, running macOS mojave 10.14.5 (18F132). 
Houdini Indie 10.0.287.

If it helps, prorender works fine through blender on this machine.

Thanks.

**Comment by hshakula** [comment]
I see why you don't see RPR in the list of renderers, for fast fix up you can do the following:
1. From terminal:
```
open /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources
```
2. Copy `Resource` and `Libraries`
3. From terminal:
```
open /Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current
```
4. Paste copied folders
5. Profit

But why do you have such issue, I don't know yet, will look into it

**Comment by hshakula** [comment]
Hm, it works just fine for me - calculates the correct installation path. What is your python version?

**Comment by axjxwright** [comment]
I have the latter.

```
→ python --version
Python 2.7.16
```

Your fix did work though, so thank you.

**Comment by axjxwright** [comment]
Just did a bit of debugging for you, and it seems that the installer is taking HFS environment variable code path:
```python
elif 'HFS' in os.environ:
    install_to_houdini_dir(os.environ['HFS'], package)
```
and my `HFS` environment variable has the `/Resources/` bit at the end. 

```
→ echo $HFS
/Applications/Houdini/Current/Frameworks/Houdini.framework/Versions/Current/Resources/
```

**Comment by hshakula** [comment]
This is totally correct and expected. The issue is here:
```
hfs = os.path.dirname(hfs)
```
Because of trailing slash it returns the same path. Fixed in 9d2bc15
